bibliosearch_backend/api/models.py

```diff
@@ -35,6 +35,8 @@ class BiblioSearchUser(models.Model):
         User,
         on_delete=models.CASCADE,
     )
+    name = models.CharField(max_length=255, default='')  
```

Comment:
set max_length to same with authors

no defaults, don't allow for empty in this field